pkg/audit/manager.go

```diff
@@ -457,7 +457,7 @@ func (am *Manager) auditResources(
 func (am *Manager) auditFromCache(ctx context.Context) ([]Result, []error) {
 	objs, err := am.auditCache.ListObjects(ctx)
 	if err != nil {
-		return nil, []error{fmt.Errorf("unable to list objects from audit cache: %w", err)}
+		return nil, []error{fmt.Errorf("unable to list objects from opa cache: %w", err)}
```

Comment:
why change `audit` to `opa` here?

Reply:
Good question! Because audit has multiple cache. So to be more exact, audit-from-cache is audit using the opa cache instead of calling the api server. Hope that helps! Do you have any other comments or is this good to merge?

Reply:
pedantic nit: audit-from-cache now uses the informer cache, since we can't currently iterate over the OPA cache

Not a blocker from my view

Reply:
Forgot about that update and just realized we need to update all the docs too. Will open a PR for that later. @maxsmythe @sozercan @acpana PTAL